Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,25 +1,78 @@
-function setAlarm() {}
+var audio = new Audio("alarmsound.mp3");
 
-// DO NOT EDIT BELOW HERE
+function setAlarm() {
+  const time = parseInt(document.getElementById("alarmSet").value); // Fetch the value of the input in seconds
```

Comment:
Keep comments consistent, either the line above or next to the line. Keep things consistent